package.json

```diff
@@ -93,5 +94,14 @@
       "^.+\\.html$": "<rootDir>/jest/htmlTransform.js"
     },
     "verbose": true
+  },
+  "standard": {
+    "env": {
+      "jest": true
+    },
+    "ignore": [
+      "**/*.spec.js",
```

Comment:
So you're not running standard on the tests?

Reply:
I can enable it for them, but initial run through them had way more errors than I was willing to fix. Most of it was things like not handling promise rejections, which we rarely ever do in a test.